### DISCUSSION TITLE
Is it possible to use floating edges which have labels on them?

### DISCUSSION THREAD

**Discussion by mrsci99** [xyflow/xyflow#4412] [Q&A]
I've been through the documentation and the examples of floating edges don't have labels showing on the edges.

**Comment by moklick** [comment]
yes, that's possible. A floating edge is a custom edge. You could add an `EdgeLabelRenderer` to render a label as seen in this example https://reactflow.dev/examples/edges/custom-edges . The button edge renders a button, but it could be a label too.
